database/migrator.go

```diff
@@ -16,20 +15,17 @@ import (
 )
 
 func RunMigrations(log log.Logger, db *sql.DB, config DatabaseConfig) error {
-	if _, err := os.Stat(config.migrationsDir); os.IsNotExist(err) {
-		return fmt.Errorf("migrations directory=\"%s\" does not exist", config.migrationsDir)
-	}
-
 	log.Info().Log("Running Migrations")
-	_ = pkger.Include(config.migrationsDir)
+
+	_ = pkger.Include("/migrations/")
```

Comment:
Does this mean all projects need this directory? Do we need to remove it from projects to prevent double imports?

See: https://github.com/moov-io/access/blob/master/package.go#L10

Reply:
In reality this Include in runtime does nothing (returns the same accepted string).

This line works only if we run `pkger` binary inside this project. It will scan all files for "pkger.Include(...)" and pack found files into pkged.go.

When we import this package and in Customers (or any other project) and run `pkger` binary it will not scan this specific file (moov/base/database/migrator.go) and will not pack anything.

That's why we have to create kind of fake file like the file you mentioned (https://github.com/moov-io/access/blob/master/package.go#L10) to let pkger know what we want him to package.

In this PR I decided to make it different. I added following to makefile:

```
build:
   pkger -include /migrations
   go build ....
```

it seems instead of passing paths via -include option I need to follow our convention and create `package.go` file the same as in access project.

Reply:
I'm worried about us requiring `pkger` being on a developers `PATH` in order to build PayGate. Sure, to create a docker image it makes sense, but even to run `go test` or `go run` type behaviors (via `make build`) we need `pkger` installed now?

Reply:
We solved this by including the generated file in our OSS projects.